lib/ansible/galaxy/data/community/.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,133 @@
+# README FIRST
```

Comment:
@Ompragash this needs to be synchronized with the recent version of https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml again.